tests/nnfw_api/src/GenModelTests/one_op_tests/RmsNorm.test.cc

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2024 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "GenModelTest.h"
+
+TEST_F(GenModelTest, OneOp_RmsNorm)
+{
+  CircleGen cgen;
+  uint32_t gamma_buf = cgen.addBuffer(std::vector<float>{1});
```

Comment:
Unfortunately, `cgen.addBuffer` does not copy `std::vector`. It just use data in a `std::vector` object. So we shouldn't pass a temporary object to it.
```suggestion
  std::vector<float> gamma_data{1};
  uint32_t gamma_buf = cgen.addBuffer(gamma_data);
```

Reply:
> Unfortunately, `cgen.addBuffer` does not copy `std::vector`. It just copies data pointer of a `std::vector` object. So we shouldn't pass a temporary object to it.

Yes, cgen.addBuffer just copies data pointer, 
But, I tracked the code further and it seems that it copies memory and makes no issue.
Please let me know if I'm wrong.
For safety, your suggestion is good. :)

Reply:
You are right. I also tracked the code and found flatbuffer makes space and copies data of the passed pointer.
Sorry for the incorrect suggestion.